tests/system/test_kfp.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import pytest
+from kfp import dsl
+
+import mlrun
+import tests.system.base
+from mlrun import mlconf
+
+
+@tests.system.base.TestMLRunSystem.skip_test_if_env_not_configured
+class TestKFP(tests.system.base.TestMLRunSystem):
+    project_name = "kfp-system-test"
+
+    @pytest.mark.enterprise
+    def test_kfp_with_mount(self):
+        code_path = str(self.assets_path / "kfp_with_mount.py")
+        kfp_with_v3io_mount = mlrun.code_to_function(
+            name="my-kfp-with-mount",
+            kind="job",
+            filename=code_path,
+            project=self.project_name,
+            image="mlrun/mlrun",
```

Comment:
can be re removed, no?

Reply:
looks like it is not being enriched so needs to be set explicitly

Reply:
should be enriched server side

Reply:
image is being enriched when `auto_build` option is enabled but it is currently not available with pipelines. will add it in a follow-up PR.